src/onnx/parse_instancenorm.cpp

```diff
@@ -65,14 +82,29 @@ struct parse_instancenorm : op_parser<parse_instancenorm>
 
         std::vector<int64_t> axes(kdims);
         std::iota(axes.begin(), axes.end(), 2);
-
         auto mean = info.add_instruction(make_op("reduce_mean", {{"axes", axes}}), x);
         auto mean_bcast =
             info.add_instruction(make_op("multibroadcast", {{"out_lens", dims}}), mean);
+        auto l1 = info.add_instruction(make_op("sub"), x, mean_bcast);
+        // for the fp16, if not converting to fp32 then divide `x` and `mean` by `sqrt(n)` and take
+        // reduce_sum to calculate variance i.e.
+        // var =  reduce_sum((x/s_n - mean/s_n)^2) where s_n = sqrt(n)
```

Comment:
This method doesn't work out of the box because of this : #1775 . It requires `export MIGRAPHX_DISABLE_REDUCE_FUSION=1`.